src/templates_front/templator_front.org

```diff
@@ -170,46 +170,47 @@ __trexio_path__ = None
 ** Error handling
 
       #+NAME: table-exit-codes
-   | Macro                            | Code | Description                            |
```

Comment:
Here, all lines are different because of the width of the column. I only added the last exit code.

Reply:
Got it, thanks